html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="get">
+    <label for="email">Email</label>
+    <div><input type="text" name="email" id="email"></div>
+    <label for="why">Why is coding cool?</label>
+    <div>
+      <select name="Why is Coding Cool" id="why">
+        <option value="select">Select an Option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful.</option>
+      </select>
+    </div>
+    <br>
+    <div>
+      <label for="favorite">Favorite Language (choose one)</label>
+      <div>
+        <input type="radio" name="favorite" value="HTML">HTML
+        <input type="radio" name="favorite" value="CSS">CSS
+        <input type="radio" name="favorite" value="JavaScript">JavaScript
```

Comment:
make sure to include a label for each of these inputs

Reply:
![image](https://user-images.githubusercontent.com/20393497/186238259-154a6543-d526-4e63-a68e-22a7b6fd064d.png)
Is this syntax acceptable if the label is short?